src/EPR.SubsidiaryBulkUpload.Application/Models/BulkUpdateErrors.cs

```diff
@@ -23,4 +22,7 @@ public static class BulkUpdateErrors
 
     public const int CompaniesAlreadyBelongsToADifferentParent = 107;
     public const string CompaniesAlreadyBelongsToADifferentParentMessage = "Invalid Request. Subsidiary already has a different parent.";
+
```

Comment:
These would be better in a separate class - they aren't errors but more like NotificationStatusKeys

Reply:
done